src/algokit/core/doctor.py

```diff
@@ -89,7 +89,10 @@ def check_dependency(
         )
     if minimum_version is not None:
         try:
-            version_ok = is_minimum_version(version_output, minimum_version)
+            match = re.search(r"^\d+\.\d+\.\d+", output)
```

Comment:
two things here:
- it was previously using `version_output` because that's the unaltered output
- raise to catch is a bit of an anti-pattern